### PR TITLE
GTFS - Part II

### PR DESCRIPTION
**Pull request by jmcanterafonseca**
WIP do not review yet

### CODE REVIEW

**Comment by jmcanterafonseca** [comment]
@chicco785 please could you r? 

thanks

File: UrbanMobility/Trip/doc/spec.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
++ `shortName`: Same as GTFS `trip_short_name`.
+  + Attribute type: Property. [Text](https://schema.org/Text).
+  + Optional
+    
++ `direction`: Same as GTFS `direction_id`.
+  + Attribute type: Property. [Text](https://schema.org/Text).
+  + Allowed Values: `"0"` and `"1"` as per GTFS `direction_id`. 
+  + Optional
+    
++ `block`: Same as GTFS `block_id`.
```
Comment (chicco785) [inline UrbanMobility/Trip/doc/spec.md]: if block is not modeled, i am not sure it make much sense to keep it

**Comment by chicco785** [comment]
will you model also time stops? without that showing "real" time data of transits will be hard in my understanding

File: UrbanMobility/Route/doc/spec.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+### Properties
+
+| GTFS Field            | NGSI Attribute          | LinkedGTFS          | Comment                                                    |
+|:--------------------- |:------------------------|:------------------- |:-----------------------------------------------------------|
+| `route_short_name`      | `shortName`           | `gtfs:shortName`    |                                                            |
+| `route_long_name`       | `name`                | `gtfs:longName`     |                                                            |
+| `route_type`            | `routeType`           | `gtfs:routeType`    |                                                            |
+| `route_desc`            | `description`         | `dct:description`   |                                                            |
+| `route_url`             | `page`                | `foaf:page`         |                                                            |
```
Comment (chicco785) [inline UrbanMobility/Route/doc/spec.md]: why page?

File: UrbanMobility/Route/doc/spec.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+| `route_color`           | `routeColor`          | `gtfs:color`        | Agency's [address](https://schema.org/address). Schema.org |
+| `route_text_color`      | `routeTextColor`      | `gtfs:textColor`    |                                                            |
+| `route_sort_order`      | `routeSortOrder`      |                     |                                                            |
```
Comment (chicco785) [inline UrbanMobility/Route/doc/spec.md]: GTFS is not describing it. is it relevant?

File: UrbanMobility/Trip/doc/spec.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  + Attribute type: Property. [Text](https://schema.org/Text)
+  + Optional
+  
++ `service`: Same as GTFS `service_id`.
+  + Attribute type: Property. [Text](https://schema.org/Text)
```
Comment (chicco785) [inline UrbanMobility/Trip/doc/spec.md]: shouldn't be an uri in the end? LinkedGTFS seems to consider it as way to reference schedules.

File: UrbanMobility/Route/doc/spec.md (hunk 3 above)
Comment (jmcanterafonseca) [reply UrbanMobility/Route/doc/spec.md]: could be we don't know

File: UrbanMobility/Route/doc/spec.md (hunk 2 above)
Comment (jmcanterafonseca) [reply UrbanMobility/Route/doc/spec.md]: we have used it to resemble LinkedGTFS

File: UrbanMobility/Trip/doc/spec.md (hunk 1 above)
Comment (jmcanterafonseca) [reply UrbanMobility/Trip/doc/spec.md]: yes, I can think on something better

File: UrbanMobility/Trip/doc/spec.md (hunk 4 above)
Comment (jmcanterafonseca) [reply UrbanMobility/Trip/doc/spec.md]: need to check

Comment (jmcanterafonseca) [reply UrbanMobility/Trip/doc/spec.md]: yes, we will, at least, model all the mandatory entities in GTFS

File: UrbanMobility/Trip/doc/spec.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+| GTFS Field                | NGSI Attribute          | LinkedGTFS                  | Comment                                                    |
+|:--------------------------|:------------------------|:--------------------------- |:-----------------------------------------------------------|
+| `trip_headsign`           | `headSign`              | `gtfs:headsign`             |                                                            |
+| `trip_short_name`         | `shortName`             | `gtfs:shortName`            |                                                            |
+| `direction_id`            | `direction`             | `gtfs:direction`            |                                                            |
+| `shape_id`                | `location`              | `gtfs:shape`                | Coordinates shall be taken from `shapes.txt` feed file.    |
+| `block_id`                | `block`                 | `gtfs:block`                |                                                            |
+| `service_id`              | `service`               | `gtfs:service`              |                                                            |
```
Comment (chicco785) [inline UrbanMobility/Trip/doc/spec.md]: shouldn't be moved to relationships?

**Comment by chicco785** [comment]
lgtm - i guess service will be defined in a different pr

**Pull request merged**